src/moments.jl

```diff
@@ -43,30 +23,22 @@ function var(v::RealArray, w::AbstractWeights; mean=nothing,
     corrected = depcheck(:var, :corrected, corrected)
 
     if mean == nothing
-        varm(v, w, Statistics.mean(v, w); corrected=corrected)
+        _moment2(v, w, Statistics.mean(v, w); corrected=corrected)
```

Comment:
why not `var` with kwarg?

Reply:
We're already in `var`. Calling ourselves with the computed mean would work but do you think it would really be cleaner?

(I'd like to deprecate `_moment2` and similar functions at some point to simplify the API and implementation...)

Reply:
:)